commands/droplet_actions.go

```diff
@@ -72,15 +75,23 @@ You can use Droplet actions to perform tasks on a Droplet, such as rebooting, re
 	cmdDropletActionEnableBackups := CmdBuilder(cmd, RunDropletActionEnableBackups,
 		"enable-backups <droplet-id>", "Enable backups on a Droplet", `Enables backups on a Droplet. This automatically creates and stores a disk image of the Droplet. By default, backups happen daily.`, Writer,
 		displayerType(&displayers.Action{}))
+	AddStringFlag(cmdDropletActionEnableBackups, doctl.ArgDropletBackupPolicy, "", "", `Path to a new backup policy in JSON or YAML format. Set to "-" to read from stdin.`, requiredOpt())
```

Comment:
Same thing here. Let's use flags instead of requiring a file.

Reply:
Done in [4f6dfbd](https://github.com/digitalocean/doctl/pull/1609/commits/4f6dfbde3d120dd11df6311f8ab1ce978a98ecec)